docs/vnext/README.md

```diff
@@ -0,0 +1,80 @@
+# Napa.js vNext (0.4.x-canary)
+vNext is a version with new worker implementation. In Napa.js vNext, napa worker loads and initializes node.js context, which enables most of node.js APIs and addon support.
+
+## Features in vNext
+- Better Node.js API compatibility. Most of [Node APIs](https://nodejs.org/api/index.html) will be supported by default. See [this](./node-api-compatibility.md) for compatibility tracking.
+- Native addon support. All native addon can be loaded in worker by default, unless it's implemented non-thread-safely.
```

Comment:
add a page to list all verified addons, or known-not-support addons.

Reply:
Pay attention to examples/modules. We need to know whether they work or not before go to public, or just make it clear that native-addon is not ready for v0.4.x-peek